src/postgraphile/postgraphile.ts

```diff
@@ -45,6 +46,13 @@ type PostGraphileOptions = {
   // `json` (which causes the stack to become an array with elements for each
   // line of the stack).
   showErrorStack?: boolean,
+  // Enables ability to modify errors before sending them down to the client
+  // optionally can send down custom responses
+  handleErrors?: ((
+    errors: Array<GraphQLErrorExtended>,
```

Comment:
As above, this should be `Array<GraphQLError>` - handling the unmangled errors

Reply:
We should add validation that throws an error if `handleErrors` is set at the same time as either `showErrorStack` or `extendedErrors`.

Reply:
@benjie where thouls the validation be created?

Reply:
Here's an example of a validation that just outputs a warning:

https://github.com/graphile/postgraphile/blob/d0283d5535d542838af9a69c16889cceb46a8f81/src/postgraphile/postgraphile.ts#L156-L162

Putting it around there would make sense to me. This one should throw though 👍

Reply:
@benjie how's this?

``` ts
  if (options.handleErrors && (options.extendedErrors || options.showErrorStack)) {
    throw Error('cannot use handleErrors with other error options')
  }
```

Reply:
throw new Error('cannot use handleErrors with other error options')